src/jsLib.gs

```diff
@@ -306,6 +306,43 @@ function removeFromArray(array, element) {
   }
 }
 
+/**
+ * Splits string into a comma separated list
+ * trims whitespace and the start and end of each element
+ * ignores empty elements
+ * @param listInString {String} String containing comma separated list
+ * @return {Array}
+ */
+function splitCommaList_(listInString) {
```

Comment:
This generic functionality was cluttering up other code so moved it into a function